llvm/test/Transforms/InstCombine/abs-intrinsic.ll

```diff
@@ -997,3 +997,99 @@ define i8 @abs_diff_sle_y_x(i8 %x, i8 %y) {
   %cond = select i1 %cmp, i8 %sub, i8 %sub1
   ret i8 %cond
 }
+
```

Comment:
Can you please add some multi-use tests? See also https://llvm.org/docs/InstCombineContributorGuide.html#add-multi-use-tests